### PR TITLE
feat(savedaddresses): update adding saved address popup to match new design

### LINKED ISSUE #13089: Update adding saved address popup to match a new design
Related Figma:
https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=21991%3A256161&mode=dev

This issue implies:
- adding selection color section
- all input field validations
- interactions within the popup
- an ephemeral notification when adding an address

Note:
Adding a new address popup should be globally accessible, cause that popup can be run, at least at this moment from 2 different contexts:
- Wallet Main section:
  - from the left tab (regardless which one is active on the right side)
  - from the tx activity details view
- Wallet Settings section

### PR DESCRIPTION
**Pull request by saledjenic**
Corresponding `status-go` PR:
- https://github.com/status-im/status-go/pull/4512

Implemented:
- adding selection color section
- all input field validations
- interactions within the popup
- an ephemeral notification when adding an address

Closes #13089

### CODE REVIEW

File: ui/imports/utils/Constants.qml [hunk 1]
```diff
@@ -679,18 +679,22 @@ QtObject {
...
...26a0\u26a1\u26a7\u26aa\u26ab\u26b0\u26b1\u26bd\u26be\u26c4\u26c5\u26c8\u26ce\u26cf\u26d1\u26d3\u26d4\u26e9\u26ea\u26f0-\u26f5\u26f7-\u26fa\u26fd\u2702\u2705\u2708-\u270d\u270f\u2712\u2714\u2716\u271d\u2721\u2728\u2733\u2734\u2744\u2747\u274c\u274e\u2753-\u2755\u2757\u2763\u2764\u2795-\u2797\u27a1\u27b0\u27bf\u2934\u2935\u2b05-\u2b07\u2b1b\u2b1c\u2b50\u2b55\u3030\u303d\u3297\u3299]|\ud83c[\udc04\udccf\udd70\udd71\udd7e\udd7f\udd8e\udd91-\udd9a\udde6-\uddff\ude01\ude02\ude1a\ude2f\ude32-\ude3a\ude50\ude51\udf00-\udf21\udf24-\udf93\udf96\udf97\udf99-\udf9b\udf9e-\udff0\udff3-\udff5\udff7-\udfff]|\ud83d[\udc00-\udcfd\udcff-\udd3d\udd49-\udd4e\udd50-\udd67\udd6f\udd70\udd73-\udd7a\udd87\udd8a-\udd8d\udd90\udd95\udd96\udda4\udda5\udda8\uddb1\uddb2\uddbc\uddc2-\uddc4\uddd1-\uddd3\udddc-\uddde\udde1\udde3\udde8\uddef\uddf3\uddfa-\ude4f\ude80-\udec5\udecb-\uded2\uded5-\uded7\udedc-\udee5\udee9\udeeb\udeec\udef0\udef3-\udefc\udfe0-\udfeb\udff0]|\ud83e[\udd0c-\udd3a\udd3c-\udd45\udd47-\ude7c\ude80-\ude88\ude90-\udebd\udebf-\udec5\udece-\udedb\udee0-\udee8\udef0-\udef8])((\ud83c[\udffb-\udfff])?(\ud83e[\uddb0-\uddb3])?(\ufe0f?\u200d([\u2000-\u3300]|[\ud83c-\ud83e][\ud000-\udfff])\ufe0f?)?)*/g;
```
Comment (caybro) [inline ui/imports/utils/Constants.qml:688]: `emoji`?

File: ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml [hunk 2]
```diff
@@ -69,60 +108,53 @@ StatusDialog {
...
             validators: [
                 StatusMinLengthValidator {
                     minLength: 1
-                    errorMessage: qsTr("Name must not be blank")
+                    errorMessage: qsTr("Please name your saved address")
+                },
+                StatusValidator {
+                    name: "check-for-no-emojis"
+                    validate: (value) => {
+                                  return !Constants.regularExpressions.emoj.test(value)
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml:139]:
```suggestion
                                  return !Constants.regularExpressions.emoji.test(value)
```

**Review by caybro — APPROVED** [review]
LGTM

File: ui/imports/utils/Constants.qml (hunk 1 above)
Comment (saledjenic) [reply ui/imports/utils/Constants.qml]: yes, updated, thanks

**Pull request merged**